apps/vr-tests-react-components/src/stories/Breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29623&commitId=75eacb3b72491ee8c1c2501038f04be8995e0a4b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Breadcrumb Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb Converged.size.pressed.chromium.png|5183| Changed|
|Breadcrumb Converged.size.default.chromium.png|5183| Changed|
|Breadcrumb Converged.size.hover.chromium.png|5183| Changed|
|Breadcrumb Converged.size.focused.chromium.png|5183| Changed|
|Breadcrumb Converged.divider type.default.chromium.png|0| Removed|
|Breadcrumb Converged.divider type.focused.chromium.png|0| Removed|
|Breadcrumb Converged.divider type.hover.chromium.png|0| Removed|
|Breadcrumb Converged.divider type.pressed.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>